content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -5,103 +5,65 @@ metaDescription: Learn how to seed your database using Prisma's integrated seedi
 
 <TopBlock>
 
-This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality (Preview). Seeding allows you to consistently re-create the same data in your database and can be used to:
+This guide describes how to seed your database using Prisma Client and Prisma's integrated seeding functionality. Seeding allows you to consistently re-create the same data in your database and can be used to:
 
 - Populate your database with data that is required for your application to start - for example, a default language or a default currency.
-- Provide basic data for validating and using your application in a development environment. This is particulary useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
-
-Watch the following video for an introduction to the `db seed` command:
-
-<div class="videoWrapper">
-
-<iframe
-  width="560"
-  height="315"
-  src="https://www.youtube.com/embed/2LwTUIqjbPo"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-  allowfullscreen
-></iframe>
-
-</div>
+- Provide basic data for validating and using your application in a development environment. This is particularly useful if you are using Prisma Migrate, which sometimes requires resetting your development database.
 
 </TopBlock>
 
 ## Requirements for seeding with TypeScript or JavaScript
```

Comment:
The paragraph below is also true if you use a bash script - this is fundamentally how you tell Prisma how to execute your seed script now.

So maybe something like that:

```suggestion
## Configuring your seed script
```